packages/cli/src/cmds/prune/prune.ts

```diff
@@ -46,31 +46,60 @@ function parseSize(size: string) {
 }
 
 export default {
-  command: 'prune <file> <size>',
+  command: 'prune <file>',
 
-  describe: 'Prune an appmap file down to the given size',
+  describe: 'Make an appmap file smaller by removing events',
 
   builder: (argv: Yargs.Argv) => {
     argv.option('output-dir', {
-      describe: 'Specifies the output directory. Pruned files will be written here.',
+      describe: 'Specifies the output directory',
       type: 'string',
       default: '.',
       alias: 'o',
     });
+
+    argv.option('json', {
+      describe: 'Return the exclusions as json',
+      type: 'boolean',
+    });
+
     argv.positional('file', {
       describe: 'AppMap to prune',
     });
-    argv.positional('size', {
+
+    argv.option('size', {
       describe: 'Prune input file to this size',
+      default: '15mb',
```

Comment:
By changing this from a positional argument to an option, the change is backwards-incompatible. So, I suggest leaving it as a positional argument - since there's no great reason to break compatibility.

Reply:
Specifying a size doesn't make sense when using this command with the `--filter` option, so I think we need to change the size to an option.